<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CpuMemoryEditPage</class>
 <widget class="QWidget" name="CpuMemoryEditPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelRubric">
     <property name="text">
      <string>Configure the number of virtual CPUs and startup behavior.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxVCPUs">
     <property name="title">
      <string>Virtual CPUs</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelVCPUsMax">
        <property name="text">
         <string>Maximum number of vCPUs:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxVCPUsMax">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>128</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelVCPUsAtStartup">
        <property name="text">
         <string>Initial number of vCPUs:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxVCPUsAtStartup">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>128</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelTopology">
        <property name="text">
         <string>CPU topology:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxTopology"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
